modules/nginx/sg.tf

```diff
@@ -22,7 +22,7 @@ resource "aws_security_group" "app_lb" {
   }
 
   tags = {
-    "Name" = "Load Balancer Security group"
+    "Name" = var.resource_tags["alb_sg"]
   }
```

Comment:
Find a way to merge Name tag with other tags( e.g. Provider: Terraform). This is aplicable to every **tags=**. related code